html/pfappserver/root/graph/dashboard.tt

```diff
@@ -711,5 +712,53 @@ END
                 </div>
             </div>
         </div>
+        <div class="tab-pane[% IF tab == "queue" %] active[% END %]" id="queue">
+            <div class="card">
+                <div class="card-title">
+                    <h2>[% l('Queue Counters') %]</h2>
+                </div>
+                <div class="card-block">
+                    <div class="row-fluid">
+                        <div class="span18 card">
+                            <div data-netdata="packetfence.redis.queue_stats_count"
+                            data-host="/netdata/127.0.0.1"
```

Comment:
We need to loop on all cluster members since redis is local to each one.

Reply:
Should we put in a "cluster" tab (2nd next to system) and put these charts there?

Reply:
I'm out until next week, unless an emergency is needed. This appears to be a simple front end adjustment in the dashboard template.

Can we merge and then either @cgx or I can update after?